Assignment2/Josesilva99-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,104 @@
+{
+	"@context":{
+		"st": "http://www.example.com/students/josesilvamanrriquez#",
+		"name": "st:name",
+		"middleName": "st:middleName",
+		"firstSurname": "st:firstSurname",
+		"secondSurname": "st:secondSurname",
+		"hometown": "st:hometown",
+		"age": "st:age",
+		"almaMater": "st:almaMater",
+		"nameFather": "st:nameFather",
+		"nameMother": "st:nameMother",
+		
+		"subjects": "st:subjects",
+		"nameSubject": "subjects:nameSubject",
+		"teachers": "subjects:teachers",
+		"nameTeacher":"teachers:nameTeacher",
+		"lastNameTeacher":"teachers:lastName",
+		"description":"subjects:description"

```

Comment:
These subject attribute definitions work, but semantically they aren't ok. The best way to define them might be defining a new prefix for the subject concept and then use it to define their attributes.
You don't need to change them, but remember to provide good definitions for the future.